desktop/package/linux/Haveno.AppDir/Haveno.desktop

```diff
@@ -0,0 +1,15 @@
+# If you want Haveno to appear in a Linux app launcher ("start menu"), install this by doing:
+# sudo desktop-file-install electrum-ltc.desktop
+
+[Desktop Entry]
+Comment=A decentralized monero exchange network.
```

Comment:
You should capitalise Monero here. Maybe also mention that it's P2P or that it uses Tor? Not required though.

Reply:
Fixed.